explorer/src/pages/AccountDetailsPage.tsx

```diff
@@ -209,8 +209,9 @@ export function AccountHeader({
       <div className="row align-items-end">
         {unverified && (
           <div className="alert alert-danger alert-scam" role="alert">
```

Comment:
What do you think about changing this to "alert-warning"? It's not *as* scary as the red danger banner but still impossible to ignore.

Reply:
<img width="1164" alt="Screen Shot 2022-04-27 at 11 26 22 AM" src="https://user-images.githubusercontent.com/82475023/165566829-6ecd5a54-df23-42fd-8dad-91bff2888174.png">
That pink/purple is just as hard to ignore. Let me update

Reply:
cool looks good!